testsuite/integration-arquillian/tests/other/console_no_users/src/test/java/org/keycloak/testsuite/console/pages/WelcomePageTest.java

```diff
@@ -0,0 +1,121 @@
+package org.keycloak.testsuite.console.pages;
+
+import java.net.InetAddress;
+import java.net.NetworkInterface;
+import java.net.URL;
+import java.util.Collections;
+import java.util.Enumeration;
+import java.util.List;
+import org.jboss.arquillian.graphene.page.Page;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+import org.keycloak.representations.idm.RealmRepresentation;
+import org.keycloak.testsuite.AbstractKeycloakTest;
+import org.keycloak.testsuite.auth.page.WelcomePage;
+import org.keycloak.testsuite.auth.page.login.OIDCLogin;
+
+/**
+ *
+ */
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class WelcomePageTest extends AbstractKeycloakTest {
+
+    private String CENTRAL_CI_IP_PREFIX = "10.8";
+    
+    private String HOME_IP_PREFIX = "10.10";
+    
+    @Page
+    private WelcomePage welcomePage;
+
+    @Page
+    protected OIDCLogin loginPage;
+    
+    @Override
+    public void addTestRealms(List<RealmRepresentation> testRealms) {
+        // no operation
+    }
+
+    /*
+     * Leave out client initialization and creation of a user account. We
+     * don't need those.
+     */
+    @Before
+    @Override
+    public void beforeAbstractKeycloakTest() {
+        setDefaultPageUriParameters();
+        driverSettings();
+    }
+
+    /**
+     * Attempt to resolve the floating IP address. This is where EAP/WildFly
+     * will be accessible. See "-Djboss.bind.address=0.0.0.0".
+     * 
+     * @return
+     * @throws Exception 
+     */
+    private String getFloatingIpAddress() throws Exception {
+        Enumeration<NetworkInterface> netInterfaces = NetworkInterface.getNetworkInterfaces();
+        for (NetworkInterface ni : Collections.list(netInterfaces)) {
+            Enumeration<InetAddress> inetAddresses = ni.getInetAddresses();
+            for (InetAddress a : Collections.list(inetAddresses)) {
+                String hostAddress = a.getHostAddress();
+                System.out.println(hostAddress);
+                if (hostAddress.startsWith(CENTRAL_CI_IP_PREFIX) || hostAddress.startsWith(HOME_IP_PREFIX)) {
```

Comment:
This will not work in many cases, please do not hardcode environment specific things into the tests. The following should do the trick:
    (!a.isLoopbackAddress && a.isSiteLocalAddress())